src/borg/archiver/extract_cmd.py

```diff
@@ -66,7 +67,10 @@ def do_extract(self, args, repository, manifest, archive):
 
             is_matched = matcher.match(orig_path)
             log_prefix = "+" if is_matched else "-"
-            logging.getLogger("borg.output.list").info(f"{log_prefix} {remove_surrogates(item.path)}")
+
+            # Conditionally log based on the output_list flag
```

Comment:
please avoid such comments. it does not add any information that is not obvious from the next 2 lines anway.

Reply:
Understood. Would you like me to remove it and commit again?

Reply:
Yes, please.